### PR TITLE
Add `Matrix4.createPerspective()`

### PR DESCRIPTION
**Pull request by bwkam**
I didn't find a createPerspective function (for creating perspective views) so I tried to make one, but I don't really know if that will work.

### CODE REVIEW

**Comment by bwkam** [comment]
Can someone re-run the CI?

File: src/lime/math/Matrix4.hx [hunk 1]
```diff
@@ -378,6 +379,52 @@ abstract Matrix4(Float32Array) from Float32Array to Float32Array
...
+	public function createPerspectiveZO(fovy:Float, aspect:Float, zNear:Float, zFar:Float):Void {
+		if (Math.abs(aspect - (Math.pow(2, -23))) > 0.0) {
+			var tanHalfFovy = Math.tan(fovy / 2.0);
+			this[0] = 1.0 / (aspect * tanHalfFovy); 
+			this[6] = 1.0 / (tanHalfFovy);
+			this[11] = zFar / (zNear - zFar);
+			this[12] = -1.0;
+			this[15] = -(zFar * zNear) / (zFar - zNear);
+		} else {
+			throw new haxe.Exception("aspect greater than epsilson for Float");
```
Comment (player-03) [inline src/lime/math/Matrix4.hx]: Don't think this works with Haxe 3. Maybe just throw the string without wrapping it.

Comment (player-03) [reply src/lime/math/Matrix4.hx]: Also without the `new` keyword. :)

Comment (bwkam) [reply src/lime/math/Matrix4.hx]: forgot about that lol

**Comment by bwkam** [comment]
Does that look good?

**Comment by bwkam** [comment]
Peek 2023-05-10 09-05

Here is a [demo](https://github.com/bwkam/learnopengl-haxelime/tree/main/Getting-Started/Coordinate-Systems/Rotating3DContainer)
Thanks to [@47rooks](https://github.com/47rooks) for the modifications.

**Pull request merged**